### DISCUSSION TITLE
Wemos D1 tasmota tasmota-PL.bin - no SDS011 sensor

### DISCUSSION THREAD

**Discussion by nemapl** [arendst/Tasmota#10908] [Q&A] [answered]
Hello,
 I've just flashed wemos D1 - it is last firmware tasmota-PL.bin available.
I cannot find option to choose sensor SDS011 - any guide? There are many option on the dropdown for RX GPIO but not SDS011

**Comment by sfromis** [comment]
Most sensors (including this one) needs tasmota-sensors.bin. This is not available in PL translation, if you need that, you'll have to compile your won binary.

**Comment by ascillato** [comment, marked as answer]
All Tasmota variants are explained in the docs and in the codebase at https://github.com/arendst/Tasmota/blob/development/BUILDS.md
The SDS sensor is included **ONLY** on **_tasmota-sensors.bin_**

If you want **_tasmota-sensors.bin_** in your language, you need to compile by yourself.
